### DISCUSSION TITLE
アドオン：Scratch Building

### DISCUSSION THREAD

**Discussion by kenuun** [Fun117/scratch-cloud-logs#4] [Q&A] [answered]
アドオン`Scratch Building`のログ監視機能を起動しているとたまにエラーが発生してしまいます。

**Comment by toakiryu** [comment, marked as answer]
おそらく Sctatch API へのリクエスト制限が原因です。

**Reply by kenuun** [reply]
対処法はありますか？

**Reply by toakiryu** [reply]
リクエスト制限への対処はできませんが、リクエスト制限がかかった時に再度自動的にリクエストを送信するようにすることで、解決ができるかもしれません。
